### DISCUSSION TITLE
[Detector Support]: Audio Detector

### DISCUSSION THREAD

**Discussion by thesebastianf** [blakeblackshear/frigate#13137] [Detector Support]
### Describe the problem you are having

It's hard to actually understand for many of the audio labels what would be the sound that triggers is

e.g.
- noise
- beep

Is there a way to just have an event triggered with any "sound" above a certain level. e.g. in an empty house I want to have an alert for any alarm, beep, loud bang or whatsoever. Would that be "noise"?

### Version

latest

### Frigate config file

```yaml
-
```

### docker-compose file or Docker CLI command

```yaml
-
```

### Relevant log output

```shell
-
```

### Operating system

Debian

### Install method

Docker Compose

### Object Detector

CPU (no coral)

**Comment by NickM-27** [comment]
No, currently there's no way to do that automatically. You could automate it though

**Reply by thesebastianf** [reply]
mhh automate it?
I mean I automate it in Home Assistant... but what sound label should I use to detect that "any sound at a certain level"?

**Reply by NickM-27** [reply]
you can call it whatever you want using the manual events api listening to the audio volume sensor

**Reply by thesebastianf** [reply]
ah ok  so just listening to this and if it put an automation to it:
![image](https://github.com/user-attachments/assets/a7cef96c-cbae-4326-8c81-9b1f545d56aa)

**Reply by NickM-27** [reply]
yes
